src/molecules/Select/README.md

```diff
@@ -1,17 +1,26 @@
 # Select
 
-To implement a `Select` wrapper into your project you'll need to add this import
+To implement a readonly `Select` wrapper into your project you'll need to add this import
```

Comment:
Think it makes sense to just call it a `select` unless it's a typeahead, unless you think the 'readonly' definition makes that clearer